test/functionalTest/cases/1367_time_measures/time_measure_stats.test

```diff
@@ -31,7 +31,7 @@ brokerStart CB 101,38 IPv4 -reqTimeStat
 --SHELL--
 
 #
-# 01. Initial call to GET /statistics (gives NO time stats as LAST /statistics is excluded)
+# 01. Pre-call to GET /statistics to give VALGRIND a possibility
```

Comment:
If the problem is related with timing counters, what about to just disble timing counters avoding `-requestTimeStat` thus making the test more deterministic?